zebrad/tests/acceptance.rs

```diff
@@ -695,11 +780,10 @@ fn full_sync_test(network: Network, timeout_argument_name: &str) -> Result<()> {
             SYNC_FINISHED_REGEX_TMP_STOP_EARLY,
         )
     } else {
-        tracing::info!(
-            ?network,
-            "skipped full sync test, \
-             set the {:?} environmental variable to run the test",
-            timeout_argument_name,
+        eprintln!(
+            "Skipped full sync test for {}, \
+            set the {:?} environmental variable to run the test",
+            network, timeout_argument_name,
```

Comment:
Consistency:

Can you please also fix:
https://github.com/ZcashFoundation/zebra/blob/be91ab29eeabe1b6d4fdc64653b954f260e76f1b/zebrad/tests/acceptance.rs#L1488
https://github.com/ZcashFoundation/zebra/blob/d0d611ef1986c20fa9a8f77bc8602cc45c57d805/zebrad/tests/common/lightwalletd.rs#L274

Reply:
https://github.com/ZcashFoundation/zebra/pull/4523/commits/0d232f2fe64c1dda1a3d2c010c939a85a874ac07